tests/Functional/SolrCommandsTest.php

```diff
@@ -21,15 +19,24 @@ class SolrCommandsTest extends TestCase
     /**
      * @test
      * @covers \Pantheon\Terminus\Commands\Solr\EnableCommand
+     *
+     * @group solr
+     * @group short
```

Comment:
I've noticed this command can take a long time to run
```suggestion
     * @group long
```

Reply:
@namespacebrian let's agree to put all working tests into "short" group for now (unless stated otherwise @cc @stovak )